jobserver/api/releases.py

```diff
@@ -259,6 +303,48 @@ def get(self, request, file_id):
         return serve_file(request, rfile)
 
 
+class ReviewAPI(APIView):
+    authentication_classes = [SessionAuthentication]
+
+    class ReleaseSerializer(serializers.Serializer):
+        files = FileSerializer(many=True)
+        metadata = serializers.DictField()
+        review = serializers.DictField()
+
+    def post(self, request, *args, **kwargs):
+        release = get_object_or_404(Release, id=self.kwargs["release_id"])
+        db_files = release.files.all()
+
+        serializer = self.ReleaseSerializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        payload_files = serializer.validated_data["files"]
+
+        with transaction.atomic():
+            release.metadata = serializer.validated_data["metadata"]
```

Comment:
Metadata is read only at this point - it should only be set when the release is created, as the end user sets it.